### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by exectails** [aura-project/aura#27]
We've switched to gettext style localization a while ago, but never finalized it.

**To-do**
- [x] Add parsing for `.po` files.
- [x] ~~Write a tool to create `.po` files from Aura's source code (optional, only needed if we can't make standard parsing work).~~
- [x] Add guide on how create and translate `.po` files.

I started this a while ago in [localization](https://github.com/aura-project/aura/tree/localization), but it needs some more work. The plan to use XML in NPCs #28 instead of dialog elements also affects this a little, as it changes how the localization phrases are parsed.

**Comment by Xcelled** [comment]
I'd also like to see an overload of Get that takes format string parameters. `Get("{0} dropped a {1}", creature.Name, itemName);` is much clearer than wrapping it in a format call. Plus, the localization tool could actually display hints then, so translators know that `{0}` is the placeholder for the creature name.

**Comment by exectails** [comment]
Done in https://github.com/aura-project/aura/tree/localization2. I think this issue can be closed once that is merged, as adding proper localization to the NPCs is basically part of #28. It can't be done before and should be done simultaneously.

@Xcelled It might not be such a bad idea, though I'm worried it might look a little confusing for the getters beside Get, that gettext typically supports.

```
Get("{0} test"); // "{0} test"
GetPlural("test", "tests", 1); // "test"
GetPlural("test", "tests", 2); // "tests"
GetParticular("foo", "test"); // "test"
GetParticularPlural("foo", "test", "tests", 3); // "tests"
```

If you start to stick formatting arguments to the end of a `GetParticularPlural`, it might be hard to follow, not only generally, but also for people who are already familiar with gettext.

```
GetParticularPlural("some specific npc context", "Example message {1} take takes a {2} and display it: {0}", "Example message {1} take takes some {2}s and display it: {0}", 3, "xy", "number");
```
